tests/tracer/test_tracer.py

```diff
@@ -1207,12 +1194,34 @@ def test_partial_flush_too_few_configure(self):
         self.test_partial_flush_too_few()
 
     @TracerTestCase.run_in_subprocess(
-        env_overrides=dict(DD_TRACER_PARTIAL_FLUSH_ENABLED="false", DD_TRACER_PARTIAL_FLUSH_MIN_SPANS="6")
+        env_overrides=dict(DD_TRACE_PARTIAL_FLUSH_ENABLED="false", DD_TRACE_PARTIAL_FLUSH_MIN_SPANS="6")
     )
     def test_partial_flush_configure_precedence(self):
         self.tracer.configure(partial_flush_enabled=True, partial_flush_min_spans=5)
         self.test_partial_flush()
 
+    @TracerTestCase.run_in_subprocess(
+        env_overrides=dict(DD_TRACER_PARTIAL_FLUSH_ENABLED="true", DD_TRACER_PARTIAL_FLUSH_MIN_SPANS="5")
+    )
+    def test_enable_partial_flush_with_deprecated_config(self):
+        # Test tracer with deprecated configs D_TRACER_...
+        self._test_partial_flush()
+    
+    def _test_partial_flush(self):
+        root = self.tracer.trace("root")
+        for i in range(5):
+            self.tracer.trace("child%s" % i).finish()
+
+        traces = self.pop_traces()
+        assert len(traces) == 1
+        assert len(traces[0]) == 5
+        assert [s.name for s in traces[0]] == ["child0", "child1", "child2", "child3", "child4"]
+
+        root.finish()
+        traces = self.pop_traces()
+        assert len(traces) == 1
+        assert len(traces[0]) == 1
+        assert traces[0][0].name == "root"
```

Comment:
Yes I completely missed that. I refactored `test_partial_flush()` so it's not decorated with `run_in_subprocess`

Reply:
I think I am wrong, I think my understanding of `run_in_subprocess` was wrong, from the source it looks like it marks the test case with some attributes, but doesn't directly run what is called.

Reply:
I also missed that it was a pre-existing pattern, so I think it is ok to assume it works for now... but we should chat with @Kyle-Verhoog to make sure we have the usage correct